packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -265,21 +283,27 @@ const createDefaultImplementation = ({ strapi, db, eventHub, entityValidator })
       componentsToDelete.map((compos) => deleteComponents(uid, compos, { loadComponents: false }))
     );
 
+    entitiesToDelete = await this.wrapResult(entitiesToDelete);
+
     // Trigger webhooks. One for each entity
     await Promise.all(entitiesToDelete.map((entity) => this.emitEvent(uid, ENTRY_DELETE, entity)));
 
     return deletedEntities;
```

Comment:
this return value is not wrapped

Reply:
it is intended, as what we return is the count of deleted entities, not the entities:

`{ count: deletedRows }` https://github.com/strapi/strapi/blob/0895e36738384a18808f907cd62fe103ccd29bfc/packages/core/database/lib/entity-manager/index.js#L409

Reply:
Can't wait for more typescript :sweat_smile:

Reply:
YES 😭